### DISCUSSION TITLE
Socrata CSV loading now possible in 1.1.1

### DISCUSSION THREAD

**Discussion by nifdy** [duckdb/duckdb-wasm#1877] [General]
Not sure if this was intended but in 1.1.1 we are now able to retrieve csv, json files from Socrata open data domains.  

`select * from 'https://data.wa.gov/api/views/sb4j-ca4h/rows.csv?accessType=DOWNLOAD' limit 1;`

Note: this [does not work](https://github.com/duckdb/duckdb/issues/5041) with the CLI as it is requesting a header which the Socrata solution does not allow but it appears that in WASM we are not doing that.  Thank you.  This is an extremely helpful feature and want to make sure we keep it in future versions.

**Comment by domoritz** [comment]
You mean "not" not "now" in the title?

**Comment by carlopi** [comment]
This discrepancy should be better documented, and likely eventually also offered as part of native DuckDB (behind a flag).

DuckDB-Wasm is mean to be more resilient, also due to the fact that due to HTTPS requirement not all servers might comply, so failure to perform and HEAD request do not implies failure to access the file, say via GET + Range-Requests.

That this works in DuckDB-Wasm is on purpose, and currently that DuckDB native do not support this use case is also on purpose.
